desktop/cmp/form/RadioInput.scss

```diff
@@ -0,0 +1,3 @@
+.xh-radio-input.bp3-control.bp3-inline {
+  margin-right: 20px;
+}
```

Comment:
Our default margin for inline form fields made it difficult to understand with button belonged to what label.